removeJobSchedules/README.md

```diff
@@ -0,0 +1,42 @@
+# Cleaning up job schedules
+This can be used for both aaS and on prem installations of CPD. This example will remove the schedule in all jobs in the specified project
+
+# Working with Projects
+
+This example works specifically for Projects but can be easily customized to work with Spaces.
+
+# Install cpdctl
+
+You need to install cpdctl from https://github.com/IBM/cpdctl
+
+The utility scripts that you'll use assume that cpdctl is on your search path. cpdctl is standalone so you can just copy it to a location already on your path if that is easiest.
+
+# Install jq
+
+These scripts rely on the utility jq for parsing JSON output from cpdctl
+
+https://jqlang.github.io/jq/manual/
+
+# Configure cpdctl to work with your cluster.
+
+For example, to work with CPDaaS:
+
+cpdctl config profile set cpdaas --url https://cloud.ibm.com --apikey {your API key}
+
+cpdctl config profile use cpdaas
+
+[Note: cpdaas is just the profile name, you could name it anything else you want.  You can have many of these profiles configured and can just switch to the one you want using 'cpdctl config profile use ']
+
+
+for an onprem install need to provide the URL of your installation along with the username and password of the user to perform the actions
+
+cpdctl config profile set myonprem --url https://yourinstallation.com --user {user_name} --password {password}
```

Comment:
I couldn't get this step to work.

When both `--user` and `--password` are supplied, I got this:
```
FAILED
Error executing command:
The option '--user' cannot be used together with '--username', '--password', '--apikey', or '--token-file'.
```

Reply:
Probably I've got a typo in those instructions.  Does it work for you if you use --username rather than --user?

Reply:
Seems `--username` should be used instead of `--user`. Now it brings me to a different error:
```
% cpdctl config profile set myonprem --url "https://yourinstallation.com" --username beatrice --password beatrice
FAILED
Configuration error:
Unrecognized response, verify profile URL "https://yourinstallation.com". Error message: "invalid character '<' looking for beginning of value".
```

Reply:
I think your new error is occurring because the URL is not an actual CPD cluster.  I played around specifying various non-CPD urls and got a variety of errors.  But it also worked fine when I actually specified the URL of an existing cluster.

This isn't a great user experience from cpdctl but that is a separate issue.